### PR TITLE
Update ptl-ref-release-notes.md

### PR DESCRIPTION
**Pull request by willllewellyn**

### CODE REVIEW

File: articles/portal/ptl-ref-release-notes.md [hunk 1]
```diff
@@ -19,6 +19,20 @@ toc_mdlink: ptl-ref-release-notes.md
...
+
+Feature update
+
+Details:
+
+- Display 2FA seed on setup
+
+- Remove the 2FA page when logged in
+
+- Implement SSO login (feature flagged)
```
Comment (m-digman) [inline articles/portal/ptl-ref-release-notes.md]: Feature flagged changes are not relevant to customers unless they are switched on. So in this instance I assume the "Implement SSO login" entry is not required.

Comment (willllewellyn) [reply articles/portal/ptl-ref-release-notes.md]: Removed

Comment (m-digman) [reply articles/portal/ptl-ref-release-notes.md]: For the other 2 entries can we create a sentence so when the customer reads this they understand what this means. If you look at the tickets, the wording is already there.
* Display plain text 2FA seed during setup process. Allowing customers that cannot use the QR code directly, to manually copy and paste this into desktop based password management apps.
* Security enhancement, removed the 2FA page when logged in.

Comment (willllewellyn) [reply articles/portal/ptl-ref-release-notes.md]: Changed

Comment (m-digman) [reply articles/portal/ptl-ref-release-notes.md]: Thanks

**Pull request merged**